src/applications/debt-letters/components/ErrorAlerts.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/component-library/AlertBox';
+import Telephone from '@department-of-veterans-affairs/component-library/Telephone';
+import moment from 'moment';
+
+export const ErrorMessage = () => (
+  <div
+    className="usa-alert usa-alert-error vads-u-margin-top--0 vads-u-padding--3"
+    role="alert"
+  >
+    <div className="usa-alert-body">
+      <h3 className="usa-alert-heading">
+        Information about your current debts is unavailable.
+      </h3>
+      <p className="vads-u-font-family--sans">
+        You can't view information about your current debts because something
+        went wrong on our end.
+      </p>
+      <p className="vads-u-margin-bottom--1">
+        <strong>What you can do</strong>
```

Comment:
This probably needs to be a header... `h4`, I think?

Reply:
ahh yea i had it that way before but saw other places in the app that used this so figured there was a reason so i'd do the same but you right ill update them thanks!